fontforge_font_creator/yaml_generator.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+import os
+import unicodedata
+
+
+class UnicodeRanger:
+    def __init__(self, start, stop):
+        self.current = start
+        self.high = stop
+        self.stop = stop
+        self._allowed_ranges = [
+            ('Co', 'Other', 'Private Use'),
+            ('LC', 'Letter', 'Cased'),
+            ('Ll', 'Letter', 'Lowercase'),
+            ('Lo', 'Letter', 'Other'),
+            ('Lt', 'Letter', 'Titlecase'),
+            ('Lu', 'Letter', 'Uppercase'),
+            ('Nd', 'Number', 'Decimal Digit'),
+            ('Sc', 'Symbol', 'Currency'),
+            ('Sm', 'Symbol', 'Math'),
+            ('So', 'Symbol', 'Other'),
+            ('Zs', 'Separator', 'Space'),
+        ]
+        self._included_ranges = [t[0] for t in self._allowed_ranges]
+
+    def __iter__(self):
+        return self
+
+    def __next__(self):
+        if self.current > self.stop:
+            raise StopIteration
+        else:
+            self.current += 1
+            while unicodedata.category(chr(self.current)) not in self._included_ranges:
+                self.current += 1
+                self.stop += 1
+            return self.current
+
+svg_dir = os.path.join(os.path.dirname(__file__), '..', 'osmaxx-symbology', 'OSMaxx_point_symbols', 'svg')
+
+svgs = [f for f in os.listdir(svg_dir) if f.endswith('svg')]
```

Comment:
Would `if f.lower().endswith('.svg')` be better?

* Allow different case
* require a period (`.`) before the filename extension